actionpack/lib/action_dispatch/routing/inspector.rb

```diff
@@ -45,12 +45,22 @@ def action
       end
 
       def internal?
-        controller.to_s =~ %r{\Arails/(info|mailers|welcome)} || path =~ %r{\A#{Rails.application.config.assets.prefix}\z}
+        internal_controller? || internal_asset?
       end
 
       def engine?
         rack_app.respond_to?(:routes)
       end
+
+      private
+        def internal_controller?
+          controller.to_s =~ %r{\arails/(info|mailers|welcome)}
```

Comment:
hey @ekampp, how are you keeping mate? Been a while.

Well, `%r{...}` lets you specify '/' without having to escape 'em, alternatively `%r{\arails/(info|mailers|welcome)} => /\arails\/(info|mailers|welcome)/`

However, you are correct re 'A' above needing to be `\A`.  It needs to be replaced with `%r{\Arails/(info|mailers|welcome)}`.